app/src/main/java/org/simple/clinic/overdue/AppointmentRepository.kt

```diff
@@ -179,15 +179,16 @@ class AppointmentRepository @Inject constructor(
     return appointmentDao.recordsWithSyncStatus(syncStatus)
   }
 
-  override fun setSyncStatus(from: SyncStatus, to: SyncStatus): Completable {
-    return Completable.fromAction { appointmentDao.updateSyncStatus(from, to) }
+  override fun setSyncStatus(from: SyncStatus, to: SyncStatus) {
+    appointmentDao.updateSyncStatus(from, to)
   }
 
-  override fun setSyncStatus(ids: List<UUID>, to: SyncStatus): Completable {
+  override fun setSyncStatus(ids: List<UUID>, to: SyncStatus) {
     if (ids.isEmpty()) {
       throw AssertionError()
     }
```

Comment:
This check is not enforced in all the repositories and the error message is blank. We should figure out how to enforce this in some way.